packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -3552,6 +3557,41 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
     }
   );
 
+  getExpandCellTooltip = memoize(
+    (expandCellTooltipProps: CSSProperties): ReactNode => {
+      if (expandCellTooltipProps == null) {
+        return null;
+      }
+
+      const wrapperStyle: CSSProperties = {
+        position: 'absolute',
+        ...expandCellTooltipProps,
+        pointerEvents: 'none',
+      };
+
+      const popperOptions: PopperOptions = {
+        placement: 'bottom-start',
+      };
+
+      return (
+        <div style={wrapperStyle}>
+          <Tooltip
+            key={Date.now()}
+            options={popperOptions}
+            ref={this.handleTooltipRef}
+          >
+            <div style={{ textAlign: 'left' }}>
+              Click to expand row
+              <br />
+              {ContextActionUtils.isMacPlatform() ? 'Cmd' : 'Ctrl'} + Click to
```

Comment:
```suggestion
              {ContextActionUtils.isMacPlatform() ? '⌘' : 'Ctrl+'} + Click to
```

By convention, mac should have it as the symbol and no plus sign. Windows has the plus sign. Neither have any whitespace.

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/keys-keyboard-shortcuts#special-character-names

https://developer.apple.com/design/human-interface-guidelines/inputs/keyboards/